src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
You want to change `utils_get_utf8_from_locale` to actually do something on Windows? (as weird as this question sounds :D).
My concern is we could break a lot of things which rely on being `utils_get_utf8_from_locale` a NOOP on Windows.

Maybe use `g_locale_to_utf8`?